src/components/common/YourCompany/YourCompany.vue

```diff
@@ -405,12 +444,17 @@ export default class YourCompany extends Mixins(
     return 'Unknown'
   }
 
+  /** The start date for a firm. */
+  private get businessStartDate (): string {
+    return 'Unknown'
```

Comment:
Unless this changes, we know which field this is:
https://github.com/bcgov/business-schemas/blob/6b0e8f0734a656aee10444b1ada2ed16e2368d5d/src/registry_schemas/schemas/registration.json#L38

Reply:
For sure, but not how/where it's retrieved yet. I'm assuming general `business` info will include this in it's response for a sp/gp but this schema references a Registration filing structure not exactly what we are looking at in this instance.

Reply:
ok for now